### DISCUSSION TITLE
Can't switch to MR mode in FM broadcast radio receiver

### DISCUSSION THREAD

**Discussion by tommobil** [egzumer/uv-k5-firmware-custom#514] [Q&A] [answered]
Hello,

I can't switch to MR mode in FM broadcast radio receiver. I can't do this with either the F+VFO/MR or long pressing the VFO/MR button. I didn't find anything related to this phenomenon.

Thanks for help!

**Comment by BruMcD** [comment]
It absolutely works on my UV-K5s (long press on VFO/MR to change while in FM receive mode)

Do you have/see any FM radio pre-sets programmed ? (in Chirp, settings FM radio ? )

**Reply by tommobil** [reply]
Thanks, this was the problem. I think I accidentally deleted the stations.

**Comment by egzumer** [comment, marked as answer]
You have to have stations in memory. Run an automatic search with longpress scan.

**Reply by tommobil** [reply]
Thanks!

After an automatic scan, it works fine. I forgot to check if there was anything in the memory. Earlier I had three stations set, but it's possible that I wiped the memory at some point.
